scoap3/modules/records/jsonschemas/hep.json

```diff
@@ -774,8 +774,7 @@
         "page_nr": {
             "type": "array",
             "items": {
-                "type": "string",
-                "description": "FIXME: shall it be integer perhaps?",
+                "type": "integer",
```

Comment:
This small change requires a lot of work on operations side: we should recreate the hep mapping in elasticsearch and reindex all the records in it to make them adhere to the new type. Is this necessary?

Reply:
I don't think this field is indexed in es.